zio-http/shared/src/main/scala/zio/http/endpoint/openapi/OpenAPI.scala

```diff
@@ -451,7 +451,7 @@ object OpenAPI {
 
     // todo maybe not the best regex, but the old one was not working at all
     // safe chars from RFC1738 "$" | "-" | "_" | "." | "+"
-    val validPath: Regex = """\/[\/a-zA-Z0-9\-_{}$.+]*""".r
+    private val validPath: Regex = """\/[\/a-zA-Z0-9\-_{}$.+]*""".r
```

Comment:
Not related to the main subject of this PR but I've seen that this implementation detail was public. I think we can make it private